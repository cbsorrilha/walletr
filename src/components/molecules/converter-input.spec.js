import React from 'react';
import { render } from '@testing-library/react';
import ConverterInput from './converter-input';

describe('Test the converter-input component', () => {
  it('should match the snapshots', () => {
    const component = render(<ConverterInput />);
    expect(component).toMatchSnapshot();
  });
});
