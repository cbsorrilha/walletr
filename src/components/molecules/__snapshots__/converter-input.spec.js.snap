// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the converter-input component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-ifAKCX iRPjiu"
        height="auto"
      >
        <div
          class="sc-EHOje dFTmIt"
        >
          <p
            class="sc-bdVaJa ejRlfi"
          />
        </div>
        <div
          class="sc-EHOje dBPDwy"
        >
          <div
            class="sc-htpNat fIXIku"
            direction="column"
          >
            <div
              class="sc-bxivhb fsigtZ"
            >
              <input
                class="sc-bwzfXH gdnGhC"
                name="amount-undefined"
                type="number"
                value=""
                width="13.5rem"
              />
            </div>
            <div
              class="sc-bxivhb gYEqsM"
              height=".5rem"
            />
          </div>
        </div>
        <div
          class="sc-EHOje dBPDwy"
        >
          <div
            class="sc-bxivhb jszyqE"
          >
            <p
              class="sc-bdVaJa jktShC"
            >
              ➡
            </p>
          </div>
        </div>
        <div
          class="sc-EHOje PZOKT"
        >
          <p
            class="sc-bdVaJa ejRlfi"
          >
             
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-ifAKCX iRPjiu"
      height="auto"
    >
      <div
        class="sc-EHOje dFTmIt"
      >
        <p
          class="sc-bdVaJa ejRlfi"
        />
      </div>
      <div
        class="sc-EHOje dBPDwy"
      >
        <div
          class="sc-htpNat fIXIku"
          direction="column"
        >
          <div
            class="sc-bxivhb fsigtZ"
          >
            <input
              class="sc-bwzfXH gdnGhC"
              name="amount-undefined"
              type="number"
              value=""
              width="13.5rem"
            />
          </div>
          <div
            class="sc-bxivhb gYEqsM"
            height=".5rem"
          />
        </div>
      </div>
      <div
        class="sc-EHOje dBPDwy"
      >
        <div
          class="sc-bxivhb jszyqE"
        >
          <p
            class="sc-bdVaJa jktShC"
          >
            ➡
          </p>
        </div>
      </div>
      <div
        class="sc-EHOje PZOKT"
      >
        <p
          class="sc-bdVaJa ejRlfi"
        >
           
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
